frontend/src/app/pages/exercises/exercise/shared/exercise-map/utility/popup-manager.ts

```diff
@@ -32,6 +33,15 @@ export class PopupManager {
     private currentlyOpenPopupOptions?: OpenPopupOptions<any>;
     private popupsEnabled = true;
 
+    private _currentClosingIds: UUID[] = [];
+
+    public get currentClosingIds(): UUID[] {
+        return this._currentClosingIds;
+    }
+    public set currentClosingIds(value: UUID[]) {
+        this._currentClosingIds = value;
+    }
```

Comment:
I think it would be enough to have a public getter that returns `currentlyOpenPopupOptions.closingUUIDs`. This would also prevent storing this information in two places, which could lead to inconsistent data otherwise.

Reply:
Done

Reply:
In your current solution, you don't need the `?` in `return this.currentlyOpenPopupOptions?.closingUUIDs;`, since you're checking for `undefined` in the if statement above.

If you prefer a shorter solution, you could use `return this.currentlyOpenPopupOptions?.closingUUIDs ?? [];` and omit the if statement.